Dalamud/Plugin/Services/INameplatesGui.cs

```diff
@@ -0,0 +1,21 @@
+using Dalamud.Game.ClientState.Objects.Types;
+using Dalamud.Game.Gui.Nameplates.Model;
+
+namespace Dalamud.Game.Gui.Nameplates;
+
+/// <summary>
+/// This class handles interacting with native Nameplate update events and management.
+/// </summary>
+public interface INameplatesGui
```

Comment:
I would strongly prefer `INameplateGui` as I and many others are likely going to not notice that it's plural, where as other services are singular.